### DISCUSSION TITLE
Getting undefined module errors

### DISCUSSION THREAD

**Discussion by OmegaBeam69** [ROM-builders/temporary#13276] [General]
error: packages/resources/devicesettings/Android.bp:17:1: "org.lineageos.settings.resources" depends on undefined module "SettingsLibDefaults"
ninja: build stopped: subcommand failed.
18:46:00 soong bootstrap failed with: exit status 1

Always getting such error on every compilation.

**Comment by Apon77** [comment]
what can i do?
